src/api/impl/OfflineServer.cpp

```diff
@@ -0,0 +1,25 @@
+#include "OfflineServer.hh"
+
+namespace ra {
+namespace api {
+namespace impl {
+
+Login::Response OfflineServer::Login(const Login::Request& request) noexcept
+{
+    Login::Response response;
+    response.Result = ApiResult::Success;
+    response.Username = request.Username;
+    response.ApiToken = "offlineToken";
+    return std::move(response);
+}
+
+Logout::Response OfflineServer::Logout(_UNUSED const Logout::Request& request) noexcept
```

Comment:
Not a big deal but you should probably comment out the names of unused variables.